<?xml version="1.0" ?>
<launch>

  <include file="$(find fwomni)/launch/display.launch"/>

  <rosparam file="$(find fwomni)/params/controller.yaml" command="load"/>
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/" args="Cont1_velocity_controller Cont2_velocity_controller Cont3_velocity_controller Cont4_velocity_controller joint_state_controller "/>

  <node pkg="fwomni" name="omni_control" type="omni_control" output="screen" />

</launch>